policyengine_us/variables/gov/states/nm/tax/income/deductions/nm_deductions.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class nm_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico income deductions"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NM
+
+    def formula(tax_unit, period, parameters):
+        itemized_on_federal_return = tax_unit("tax_unit_itemizes", period)
+        itm_ded = tax_unit("nm_itemized_deductions", period)
+        standard_ded = tax_unit("standard_deduction", period)
+        itemized_or_standard = where(
```

Comment:
Please include a comment indicating the source for NM residents having to take the same itemization choice as on federal return